examples/02_matrix_vector_product_basic.cpp

```diff
@@ -2,6 +2,14 @@
 
 #include <iostream>
 
+#if ! defined(__GNUC__) || __GNUC__ > 9
```

Comment:
is this correct?

Reply:
I.e. do you need parenthesis?

Reply:
I don't think so, but I'll put parens in.

Reply:
Fixed!